primitives/keystore/src/lib.rs

```diff
@@ -0,0 +1,385 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2019-2020 Parity Technologies (UK) Ltd.
```

Comment:
was this created and published last year already? nope, new file.

```suggestion
// Copyright (C) 2020 Parity Technologies (UK) Ltd.
```